api/src/test/java/org/pmiops/workbench/db/dao/UserServiceTest.java

```diff
@@ -167,4 +178,13 @@ public void testClearsEraCommonsStatus() throws Exception {
     assertThat(user.getEraCommonsCompletionTime()).isNull();
   }
 
+  @Test
+  public void testSyncTwoFactorAuthStatus() throws Exception {
+    when(directoryService.getUser(EMAIL_ADDRESS)).thenReturn(googleUser);
```

Comment:
it looks like `googleUser` is only used in this test. Should we move narrow down the scope to this function since it is the only one that needs it?